.github/workflows/ios-build.yaml

```diff
@@ -47,10 +47,9 @@ jobs:
       - name: Importing translation files
         shell: bash
         run: |
-          export PATH=`pwd`/qt_ios/6.2.3/clang_64/bin:$PATH
-          export PATH=`pwd`/qt_ios/6.2.3/ios/bin:$PATH
+          export PATH=`pwd`/qt_ios/6.2.3/clang_64/bin:`pwd`/qt_ios/6.2.3/ios/bin:$PATH
```

Comment:
we should switch the two so that `qmake` gets found in `ios` first and not in `clang_64`.